examples/convert_eeg_to_bids.py

```diff
@@ -87,7 +89,11 @@
 # MNE-Python's io module and the `read_raw_edf` function.
 data_dir = os.path.join(data_dir, 'physiobank', 'database', 'eegmmidb')
 edf_path = os.path.join(data_dir, 'S001', 'S001R02.edf')
-raw = read_raw_edf(edf_path, preload=True)
+
+raw = read_raw_edf(edf_path, preload=False, stim_channel=None)
+annot = read_annotations_edf(edf_path)
```

Comment:
@agramfort this should not be necessary in MNE ... `read_raw_edf` should work with `preload=False` since you have all the necessary components.

Reply:
also I'm a bit confused about what namespace `read_annotations_xxx` belong to. You have `mne.read_annotations` for fif files and then you have `mne.io.read_annotations_brainvision` and `mne.io.read_annotations_eeglab`. And for edf it is `mne.io.edf.edf.read_annotations_eeglab` ...

Reply:
oops, looks like docstring of this function is [also incorrect](https://github.com/mne-tools/mne-python/blob/master/mne/io/edf/edf.py#L1127)

Reply:
yes agreed. We are not consistent. Can you open an issue on mne-python tracker to discuss this?

Reply:
done!